examples/giantswarm-cluster/cluster_guppy_custom-node-names.yaml

```diff
@@ -60,7 +60,6 @@ metadata:
   namespace: org-multi-project
 data:
   values: |
-    clusterName: test-customname
```

Comment:
This one we can keep because the CM is also used by the `default-apps` app which takes clusterName (I know it's confusing but it's because it's 1 CM). Could be improved I suspect.